hw02-generics/src/main/java/homework/CustomerReverseOrder.java

```diff
@@ -0,0 +1,16 @@
+package homework;
+
+import java.util.ArrayDeque;
+
+public class CustomerReverseOrder {
+
+    private final ArrayDeque<Customer> arrayDeque = new ArrayDeque<>();
```

Comment:
Лучше тип слева объявлять как интерфейс:

```suggestion
    private final Deque<Customer> arrayDeque = new ArrayDeque<>();
```